ui/raidboss/data/06-ew/raid/p2n.txt

```diff
@@ -0,0 +1,66 @@
+### P2N: Asphodelos: The Second Circle
+#
+# -ii 6808 680A 6801 680C
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 100000 jump 0
+
+0.0 "--sync--" sync /Engage!/ window 0,1
+9.3 "--sync--" sync / 14:[^:]*:Hippokampos:680F:/ window 20,20
+14.3 "Murky Depths" sync / 1[56]:[^:]*:Hippokampos:680F:/
+26.6 "Doubled Impact" sync / 1[56]:[^:]*:Hippokampos:680E:/
+32.8 "--middle--" sync / 1[56]:[^:]*:Hippokampos:6835:/
+43.6 "Spoken Cataract" sync / 1[56]:[^:]*:Hippokampos:(67F7|67F8|67F9|67FD):/
+58.3 "Spoken Cataract" sync / 1[56]:[^:]*:Hippokampos:(67F7|67F8|67F9|67FD):/
+72.0 "Murky Depths" sync / 1[56]:[^:]*:Hippokampos:680F:/
+77.2 "--middle--" sync / 1[56]:[^:]*:Hippokampos:6835:/
+84.6 "Sewage Deluge" sync / 1[56]:[^:]*:Hippokampos:67F6:/
+111.0 "Tainted Flood" sync / 1[56]:[^:]*:Hippokampos:6809:/
+126.9 "Predatory Sight" sync / 1[56]:[^:]*:Hippokampos:680B:/
+137.3 "Shockwave" sync / 1[56]:[^:]*:Hippokampos:6807:/
+148.6 "Dissociation" sync / 1[56]:[^:]*:Hippokampos:6804:/
+153.7 "--sync--" sync / 1[56]:[^:]*:Hippokampos:6805:/
+165.8 "Dissociation Dive" sync / 1[56]:[^:]*:Hippokampos:6806:/
+172.5 "--sync--" sync / 1[56]:[^:]*:Hippokampos:6D14:/
+178.2 "Coherence Flare" sync / 1[56]:[^:]*:Hippokampos:6800:/
+180.7 "Coherence Line" sync / 1[56]:[^:]*:Hippokampos:6802:/
+182.7 "--sync--" sync / 1[56]:[^:]*:Hippokampos:6803:/
+194.8 "Murky Depths" sync / 1[56]:[^:]*:Hippokampos:680F:/
+
+201.0 "--middle--" sync / 1[56]:[^:]*:Hippokampos:6835:/
+208.6 "Sewage Deluge" sync / 1[56]:[^:]*:Hippokampos:67F6:/
+227.1 "Tainted Flood" sync / 1[56]:[^:]*:Hippokampos:6809:/
+235.0 "Spoken Cataract" sync / 1[56]:[^:]*:Hippokampos:(67F7|67F8|67F9|67FD):/
+250.7 "Sewage Eruption 1" sync / 1[56]:[^:]*:Hippokampos:680D:/
+252.2 "Sewage Eruption 2" sync / 1[56]:[^:]*:Hippokampos:680D:/
+253.7 "Sewage Eruption 3" sync / 1[56]:[^:]*:Hippokampos:680D:/
```

Comment:
```suggestion
250.7 "Sewage Eruption 1" #sync / 1[56]:[^:]*:Hippokampos:680D:/
252.2 "Sewage Eruption 2" #sync / 1[56]:[^:]*:Hippokampos:680D:/
253.7 "Sewage Eruption 3" #sync / 1[56]:[^:]*:Hippokampos:680D:/
```

Here and elsewhere, these syncs need to be commented out, as they are too close together. Otherwise this happens:
```
345.500: Matched entry: 344.0 Sewage Eruption 2 (-1.500s)
345.500: Matched entry: 345.5 Sewage Eruption 3 (+0.000s)
345.500: Matched entry: 344.0 Sewage Eruption 2 (-1.500s)
345.500: Matched entry: 345.5 Sewage Eruption 3 (+0.000s)
345.500: Matched entry: 344.0 Sewage Eruption 2 (-1.500s)
345.500: Matched entry: 345.5 Sewage Eruption 3 (+0.000s)
```

Reply:
I see, they are within1.5s where that is possible. I wasn't seeing this, but I'll update that.